src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -291,6 +313,107 @@ static TryCatchStatement composeUnaryRpcMethodSampleCode(
         .build();
   }
 
+  @VisibleForTesting
+  static TryCatchStatement composeUnaryPagedRpcMethodSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames,
+      TypeNode repeatedResponseType) {
+    VariableExpr clientVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder()
+                .setName(JavaStyle.toLowerCamelCase(clientType.reference().name()))
+                .setType(clientType)
+                .build());
+    // List of rpc method arguments' variable expressions.
+    List<Expr> rpcMethodArgVarExprs =
```

Comment:
With this, we've established that the logic to do `List<MethodArgument> -> List<VariableExpr>` is used repeatedly, and is fairly generic/common enough to abstract into a separate method.

Reply:
sg